src/openeo_grass_gis_driver/actinia_processing/base.py

```diff
@@ -462,6 +485,10 @@ def openeo_to_actinia(node: Node) -> Tuple[list, list]:
                 pc["inputs"].append(param)
                 output_objects.append(output_object)
                 node.add_output(output_object=output_object)
+        if module_name in T_BASENAME_MODULES_LIST:
+            param = {"param": "basename",
+                     "value": output_object.grass_name()}
```

Comment:
This should be safe as the generation of `output_object.grass_name()` includes a uuid. But would it make sense to add the string `basename` maybe behind `output_object.grass_name()`? In that case it is more clear where the maps come from.

Reply:
I used `output_object.grass_name()` as is as basename because a suffix is appended to basename by the respective modules anyway, and with `basename = output_object.grass_name()` it is IMHO already clear where the maps come from and belong to. Also, the names can already become quite long, therefore I hesitate to make them even longer.

Reply:
Moreover, the `basename` parameter is hidden from openeo users, thus including the term `basename` is probably not very informative.

Reply:
:+1: